games/widelands/Portfile

```diff
@@ -3,6 +3,8 @@
 PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           boost 1.0
+PortGroup           github 1.0
+PortGroup           app 1.0
 
 cmake.build_type    Release
```

Comment:
does it work when not setting this? In that case it will default to the "MacPorts" type as set by the cmake PG and presumably uses the flags set by MAcPorts.

Reply:
The CMakeLists logic seems to assume that the configuration will be either `Release` or `Debug`, so I don't think it's worth touching.

Reply:
I am not sure that this is actually the case to me the logic in [CMakeLists](https://github.com/widelands/widelands/blob/59494d191032bd961327fb7e4452ac3d09383485/CMakeLists.txt#L178) checks whether `CMAKE_BUILD_TYPE` is not set or set to an empty string - removing the `cmake.build_type` here will set it to `MacPorts` so I don't think that is an issue. Please do give that a try and if it actually doesn't work then we can keep it as is, but I'm not convinced about that yet.

Reply:
When removing the line in question, configuration fails with

```
:info:configure CMake Error at CMakeLists.txt:213 (message):
:info:configure   Unknown CMAKE_BUILD_TYPE: MacPorts
:info:configure -- Configuring incomplete, errors occurred!
```

Reply:
ah, okay - I didn't read far enough in the CMakeLists file... of course one could just remove that `else` statement and it will probably work. But again, I don't really have an interest in this and if it works for you (and the appropriate flags from MacPorts are added to the build command) then I guess it's fine with me